subsys/bluetooth/mesh/vnd/Kconfig

```
@@ -31,4 +31,41 @@ config BT_MESH_SILVAIR_ENOCEAN_AUTO_COMMISSION
 
 endif
 
+config BT_MESH_DM_CLI
+	bool "Distance Measurement Client"
+	select BT_MESH_VENDOR_MODELS
+	depends on DM_MODULE
+	help
+	  Enable Distance Measurement Client model.
+
+config BT_MESH_DM_SRV
+	bool "Distance Measurement Server"
```

```suggestion
config BT_MESH_DM_SRV
	bool "Distance Measurement Server"
	imply CONFIG_DM_MODULE=y
        imply CONFIG_NEWLIB_LIBC=y
        imply CONFIG_MPSL_TIMESLOT_SESSION_COUNT=2
        imply CONFIG_DM_INITIATOR_DELAY_US=100
        imply CONFIG_DM_REFLECTOR_DELAY_US=100
        imply CONFIG_DM_INITIATOR_RANGING_WINDOW_US=60000
        imply CONFIG_DM_REFLECTOR_RANGING_WINDOW_US=70000        
```

So that all user has to do is include the DM model, and relevant libraries are automatically included. As shown [here](https://github.com/omkar3141/sdk-nrf/blob/53fb0582da728e3b35e125bfc971c1cb8d5962d8/samples/bluetooth/mesh/light/prj.conf#L63).

Imply does not mandate the value, so if users have different values for any params,in their prj.conf, those values will be used.

---

Discussed in person